src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.data.person;
+
```

Comment:
missing header comments.
what does class block mean? how will users or developers know what this class does/what to do with this class?